sysutils/sketchybar/Portfile

```diff
@@ -5,9 +5,12 @@ PortGroup               github 1.0
 PortGroup               makefile 1.0
 PortGroup               xcodeversion 1.0
 
+version             2.12.2
```

Comment:
this is set already by the github PG - just add the version to the github.setup line instead and please move the "revision" line down there as well. 

And keep the alignment the same throughout the Portfile.